debian/control

```diff
@@ -147,6 +147,8 @@ Recommends: btrfs-tools,
             libradosstriper1 (= ${binary:Version}),
             librbd1 (= ${binary:Version}),
             ntp | time-daemon,
+            nvme-cli,
```

Comment:
```suggestion
Recommends: nvme-cli,
```

Reply:
@ideepika why?

Reply:
aah sorry, my bad didn't check few lines back, thanks @tchaikov!

Reply:
thanks, @ideepika and @tchaikov!